include/pgduckdb/pgduckdb_duckdb.hpp

```diff
@@ -15,13 +15,24 @@ class DuckDBManager {
 	static inline DuckDBManager &
 	Get() {
 		static DuckDBManager instance;
+		if (!instance.database) {
+			instance.Initialize();
```

Comment:
Perform initialization out of constructor so we don't throw during object creation.